pom.xml

```diff
@@ -17,9 +17,9 @@
   <url>https://github.com/jenkinsci/winstone</url>
 
   <properties>
-    <revision>5.12</revision>
+    <revision>5.11.1</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jetty.version>9.4.32.v20200930</jetty.version>
+    <jetty.version>9.4.30.v20200611</jetty.version>
```

Comment:
I thought we were at 9.4.31? Or was this the last version used in 5.11?

Reply:
5.10 was 9.4.30.v20200611 see https://github.com/jenkinsci/winstone/blob/2bd71ffedba612833704d734e1323223e52a23d7/pom.xml#L22